src/settings/patronBlocks/Limits/LimitsForm.js

```diff
@@ -18,28 +19,55 @@ import {
 } from '@folio/stripes/components';
 import stripesFinalForm from '@folio/stripes/final-form';
 
-import { feeFineBalanceId } from '../../../constants';
+import {
+  feeFineBalanceId,
+  recallOverdueId,
+} from '../../../constants';
 
 import css from '../patronBlocks.css';
 
 function validation(value, type) {
   const numberValue = toNumber(value);
-  const min = 0.01;
-  const max = 9999.99;
+  let min;
+  let max;
+
+  if (type === 'limits') {
```

Comment:
@annamelnyk I wonder if `validation` could take `min` and `max` as params instead of `type`. That way you could avoid having these `if` statements. What do you think?

Reply:
Totally agree. Thanks @mkuklis for review and fresh look.